tests/integration/yatai_server/test_bento_repository_api.py

```diff
@@ -15,12 +13,6 @@
 
 logger = logging.getLogger('bentoml.test')
```

Comment:
@yubozhao we should avoid using docker in unit tests - moving this to the integration tests suite